test/NonfungibleTokenPositionDescriptor.spec.ts

```diff
@@ -299,11 +301,11 @@ describe('NonfungiblePositionManager', () => {
 
       describe('when token1 has more precision decimals than token0', () => {
         it('returns the correct string when the decimal difference is even', async () => {
-          expect(await nftDescriptor.fixedPointToDecimalString(ratio, 10, 18)).to.eq('0.0000000099999')
+          expect(await nftDescriptor.fixedPointToDecimalString(ratio, 10, 18)).to.eq('0.00000001')
         })
 
         it('returns the correct string when the decimal difference is odd', async () => {
-          expect(await nftDescriptor.fixedPointToDecimalString(ratio, 7, 18)).to.eq('0.00000000099999')
+          expect(await nftDescriptor.fixedPointToDecimalString(ratio, 7, 18)).to.eq('0.000000001')
```

Comment:
_technically_ 5 sig figs would be `0.0000000010000` but this is ok too

https://www.wolframalpha.com/input/?i=0.01+with+5+sig+figs

Reply:
yah, I just thought it was ugly. maybe I'll leave it though, because truncating 0's after the decimal is more code....and.......there's already a lot of code here.

Reply:
if it's more code to not print the zeroes, i would definitely just leave them

Reply:
^-- yah that's what i was saying, but I think it looks like I was saying the opposite lol